plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile

```diff
@@ -445,31 +445,30 @@ p=. Orthogonal states
 
 h3(#sclang_shallow_history_states). Shallow history states
 
-A *shallow history state* is a pseudo state that is placed inside the region of a composite state. It is used to remember the last active state inside a composite state. This makes it possible to jump back to the remembered state instead of starting at the inner initial state again.
+A *shallow history state* is a pseudo state that is placed inside the region of a composite.
```

Comment:
I do prefer the old version. The new one is confusing.

Reply:
I restored the removed phrase as it tells the reader what a shallow history state is good for. It is a necessary step on the ladder of understanding and thus should not be left out.